ui/apps/everest/src/pages/databases/expandedRow/ExpandedRow.tsx

```diff
@@ -52,6 +52,7 @@ export const ExpandedRow = ({
     port,
     raw,
   } = row.original;
+  console.log('row.original', row.original);
```

Comment:
console.log ☝️

Reply:
probably we don't have that rule, better to add it in a PR 🤔